.travis.yml

```diff
@@ -91,8 +111,28 @@ matrix:
         apt:
           sources:
             - ubuntu-toolchain-r-test
+            - llvm-toolchain-trusty-3.9
           packages:
             - g++-6
+            # TODO: find a smaller subset of these LLVM packages needed
+            - clang-3.9
+            - clang-3.9-doc
+            - libclang-common-3.9-dev
+            - libclang-3.9-dev
+            - libclang1-3.9
+            - libclang1-3.9-dbg
+            - libllvm-3.9-ocaml-dev
+            - libllvm3.9
+            - libllvm3.9-dbg
+            - lldb-3.9
+            - llvm-3.9
+            - llvm-3.9-dev
+            - llvm-3.9-doc
+            - llvm-3.9-examples
+            - llvm-3.9-runtime
+            - clang-format-3.9
+            - python-clang-3.9
+            - libfuzzer-3.9-dev
```

Comment:
I'd expect all we need is `llvm-3.9-dev`.  However, we need to make sure it is 3.9.1, not 3.9.0.